scripts/getversion.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+	echo >&2 "Usage: getversion.sh <directory>"
+	exit 1
+fi
+
+cd "$1" || exit 1
+
+cat .scmversion 2>/dev/null && exit 0
```

Comment:
I would suggest something like this:

```suggestion
if [ "$2" -ne "--ignore-scmversion" ]; then
  cat .scmversion 2>/dev/null && exit 0
fi
```

So you can do `./scripts/getversion.sh . --ignore-scmversion > .scmversion` in your meta build script.

Reply:
`rm -f .scmversion;` is shorter than `--ignore-scmversion` :)